config/babel.dev.js

```diff
@@ -9,38 +9,66 @@
  */
 // @remove-on-eject-end
 
+
+// After eject
+module.exports = function () {
```

Comment:
This file existing in a trimmed down state is somewhat confusing. Can we embed this single option into the Webpack config and instead branch there? You could use `Object.assign()` to merge options if necessary.